README.md

```diff
@@ -9,11 +9,14 @@ Role Variables
 --------------
 
 ```yaml
-java_version: 9.0.7
-java_zulu_mirror: https://cdn.azul.com/zulu/bin
-java_zulu_build: 1
+java_version: 11.2.3
 ```
 
+Typically, using the default `java_version` is recommended (it will be updated by Humio staff as needed), but you can
+find valid versions by taking a look [here](http://static.azul.com/zulu/bin/). Note that the role will automatically
```

Comment:
["Here" links](https://www.w3.org/QA/Tips/noClickHere) are a bad practice.

The link should be https too, `https://static.azul.com/zulu/bin/`